### PR TITLE
Subscription feature

### PR DESCRIPTION
**Pull request by kmthorsnes**
#### Description
Changes proposed in this pull request:

* Adds the possibility to buy a subscription
* Adds stripe as a payment-gateway
#### What I have learned working on this feature: 
To work with the Stripe-API

#### Screenshots:
skarmavbild 2018-03-17 kl 12 16 23

### CODE REVIEW

**Review by davidboll — COMMENTED** [review]
Looks good to me

File: config/routes.rb [hunk 1]
```diff
@@ -2,6 +2,8 @@
   root controller: :home, action: :index
 
   devise_for :users
-
+  resources :subscription, only: [:new]
   resources :articles, only: [:new, :index, :create, :show]
+  root controller: :home, action: :index
+  post '/charges', controller: :subscription, action: :create
```
Comment (magnus-thor) [inline config/routes.rb]: delete this line and update `resources :subscription, only: [:new, :create]` and in your view use the post route for subscriptions

File: features/step_definitions/subscribe_steps.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  within_frame @stripe_iframe do
+    fill_in field, with: content
+  end
+end
+
+And(/^submit the stripe form$/) do
+  within_frame @stripe_iframe do
+    find('.Section-button').click
+  end
+  sleep(10)
```
Comment (magnus-thor) [inline features/step_definitions/subscribe_steps.rb]: check if you need this to be so long a sleep

**Review by magnus-thor — APPROVED** [review]
Good job.

File: config/credentials.yml.enc [hunk 3]
```diff
@@ -1 +1 @@
-DNZ9DfwE/rVU39g9r9bHkmftW5RQKtYvsUOYcGTrIdR6Oydnvl3CxOLTgimKg++NaqgTeFInZEXyAsdeDkX87mSBIrw2C3p/h66on8PZjj/K4pW6tQmwD+Rm4r2ugT38XIIFQVRtJW/IJ6oy1r9UqohnC5RwWoJ+gOiyHjZ5dwG3W5pGZUX5VFbEn3MmcjQpn3eNqj3hAlUTjPIaFNNUJB9k/vR2f4ESrWPAMagw3gVsa/R0Qr+olUYYy+OexwBg3gKE7ncgWxP7XtE7NHzO+j5uf1tRiVV9R6NJ3SeVFZ9AXL4rj+kadIbK8FaA1RQVNoJBKhyg2wA1ujJalNPDuDBKo+0wMtjAAIxCCOFMeBKR829+J1+2eQ7TDHQ1XhBAe2AM3PJ+hZ+vCnA0HlLmneDd73HlocJO--0bpCoyhkvgi77A4m--EtQqzNJsFPkH0vSQ0LsZOA==
\ No newline at end of file
+zOWq3CLlhFO4Z2vr64K6ghG2/8cgBgY6XKP5llITmdESTTYEvzJJe4R8dpGz/Q3s6mdTzTtRlQG3aF1cnTaOHCS8ZP5EFrNQXd36M2jfcf9Ldjj6snHg8SBtsyZXpg/0Vj6PrAhW0khXuC5KZ86U8q881JcGk5foA5NAsvhJVOcz4TxxstAV8yQ4z6RoBlMvuEu/MncJTpiO3hp8iXXkHyg5Ji/2PNguwqCZcFW/aINH3ghuhVdD5iMIUyk5gE9xcekYF1yvThlqu2R28ECzPMIdUcsDpbJwpVUk4HzyNl2cmANjYvL1IA9mYQczWeZNGZA87AArd6MheEMCIH27Yo+vzbPnvO6zVX2wOKrgeaPs6lWP3S5IL3CAkhyOKlleISAnh5Z1GnQdacDWrLWhW1GHlCYiNPAHIUbM/Ff6C/TnaxIzTqY7p/uaQKDN81QeXQ4eH4PUXSK/ih+2NV7Ni1l27Vu+OfhS2LhDUe71kSE17/kI7PQeqKU17JYtkQz2iE1m5BAx+oddPfeY0TzULYeut5/RFWds72GQ180=--6eWXinmVgYITSYFD--PY0SBxtlgqAN0SyV1LcFpg==
```
Comment (AKidd95) [inline config/credentials.yml.enc]: should this be included in version control?

**Review by AKidd95 — APPROVED** [review]
looks good to me

**Review by AntonellaMorittu — APPROVED** [review]

**Comment by diraulo** [comment]
Why is this still not merged if it was approved more than an hour ago? 🤔

**Pull request merged**